src/engine/controls/loopingcontrol.cpp

```diff
@@ -958,6 +959,14 @@ void LoopingControl::updateBeatLoopingControls() {
 }
 
 void LoopingControl::slotBeatLoop(double beats, bool keepStartPoint, bool enable) {
+
+    // if a seek was queued in the engine buffer move the current sample to its position
+    ControlValueAtomic<double> seekPosition;
+    if(getEngineBuffer()->isSeekQueued(seekPosition)){
```

Comment:
Please use git-clang-format HEAD~1 to fix some code style issues. 
https://www.mixxx.org/wiki/doku.php/coding_guidelines#command_line